pyproject.toml

```diff
@@ -48,7 +49,8 @@ black = ">=23.3.0"
 isort = ">=5.12.0"
 openpyxl = ">=3.0.10"
 # for tables, MacOS gives random CI failures on 3.9.2
-tables = { version = "==3.10.1", python = "<4" }   # 3.8.0 depends on blosc2 which caps python to <4
+# for tables, Windows build fails on 3.10.1
+tables = { version = ">=3.10.1", python = "<4" }  # 3.8.0 depends on blosc2 which caps python to <4
```

Comment:
remove the comment and make it `>= 3.10.2` since that is what worked for Windows with 3.13 anyway and 3.10.2 is needed with python 3.13.